analytic_engine/src/sst/parquet/async_reader.rs

```diff
@@ -54,9 +55,10 @@ type SendableRecordBatchStream = Pin<Box<dyn Stream<Item = Result<ArrowRecordBat
 pub struct Reader<'a> {
     /// The path where the data is persisted.
     path: &'a Path,
-    hybrid_encoding: bool,
     /// The storage where the data is persist.
     store: &'a ObjectStoreRef,
+    /// The hint for the sst file size. One io will be avoided if provided.
```

Comment:
```
One io will be avoided if provided.
```
It seems this comment is not suitable here, a better place could be in `ReadHint`